parent/pom.xml

```diff
@@ -493,6 +493,20 @@
             <version>${version.hibernate.entitymanager}</version>
          </dependency>
          <dependency>
+            <groupId>org.jboss.shrinkwrap.resolver</groupId>
+            <artifactId>shrinkwrap-resolver-bom</artifactId>
+            <version>${version.shrinkwrapResolver}</version>
+            <type>pom</type>
+            <scope>import</scope>
+         </dependency>
+         <!-- This is a work around for https://issues.jboss.org/browse/SHRINKRES-238 -->
+         <!-- After fixing the issue above, the dependency below can be safely removed -->
```

Comment:
How about the `import` dependency above? Would you still need it if shrinkwrap resolved transitive dependencies properly?